README.md

```diff
@@ -118,11 +118,30 @@ or a specific version:
 
     docker run -d -e VERSION=1.7.9 ...
 
-When using "LATEST" or "SNAPSHOT" an upgrade can be performed by simply restarting the container. 
+When using "LATEST" or "SNAPSHOT" an upgrade can be performed by simply restarting the container.
 During the next startup, if a newer version is available from the respective release channel, then
 the new server jar file is downloaded and used. _NOTE: over time you might see older versions of
 the server jar remain in the `/data` directory. It is safe to remove those._
 
+## Running Minecraft server on different Java version
+
+To use a different version of Java, please use a docker tag to build your Minecraft server.
```

Comment:
To avoid confusion with `docker build` process, I suggest avoiding the word "build" here:
```suggestion
To use a different version of Java, please use a docker tag to run your Minecraft server.
```